h/templates/activity/search.html.jinja2

```diff
@@ -82,6 +82,14 @@
 
   {{ panel('navbar') }}
 
+  {% for url in asset_urls("search_js") %}
+    <script src="{{ url }}"></script>
```

Comment:
I will add defer to this script as it is not required that we have it immediately